library/build.gradle

```diff
@@ -6,11 +6,10 @@ plugins {
 
 android {
   namespace 'com.mux.video.upload'
-  compileSdk 34
+  compileSdk 35
 
   defaultConfig {
     minSdk 16
-    targetSdkVersion 34
```

Comment:
Does this affect anything?

Reply:
Not for us.

For apps, it's pretty meaningful but libraries would have their `targetSdkVersion` overridden by the final app build.